simulation/templates/template_unit_support_female_citizen.xml

```diff
@@ -45,11 +42,18 @@
       structures/{civ}_storehouse
       structures/{civ}_farmstead
       structures/{civ}_field
-      structures/{civ}_corral
       structures/{civ}_dock
       structures/{civ}_temple
       structures/{civ}_market
       structures/{civ}_civil_centre
+      structures/{civ}_outpost
+      structures/{civ}_wallset_wood
+      structures/{civ}_barracks
+      structures/{civ}_blacksmith
+      structures/{civ}_defense_tower
+      structures/{civ}_wallset_stone
+      structures/{civ}_fortress
+      structures/{civ}_wonder

```

Comment:
@LordGood had added those buildings to the build lists of the respective `{earth|pega|uni}_support_female_citizen.xml` files, so as unusual as it seems, it appears so.